tests/testthat/test-qenv_eval_code.R

```diff
@@ -36,21 +36,27 @@ testthat::test_that("getting object from the package namespace works even if lib
 testthat::test_that("eval_code works with character", {
   q1 <- eval_code(qenv(), "a <- 1")
   testthat::expect_identical(get_code(q1), "a <- 1")
-  testthat::expect_equal(q1@.xData, list2env(list(a = 1)))
+  testthat::expect_equal(q1, list2env(list(a = 1)))
 })
 
 testthat::test_that("eval_code works with expression", {
-  q1 <- eval_code(qenv(), as.expression(quote(a <- 1)))
-
-  testthat::expect_identical(get_code(q1), "a <- 1")
-  testthat::expect_equal(q1@.xData, list2env(list(a = 1)))
+  q1 <- eval_code(qenv(), expression(a <- 1, b <- 2))
+  testthat::expect_identical(get_code(q1), "a <- 1\nb <- 2")
+  testthat::expect_equal(q1, list2env(list(a = 1, b = 2)))
+testthat::test_that("eval_code preserves original formatting when `srcref` is present in the expression", {
```

Comment:
Here there is a typo, the test "eval_code works with expression" should be closed after line 45:

```suggestion
})

testthat::test_that("eval_code preserves original formatting when `srcref` is present in the expression", {
```